### DISCUSSION TITLE
did someone deployed this wonderfull project in K8S/K3S ?

### DISCUSSION THREAD

**Discussion by Alulu2401** [zoriya/Kyoo#694] [General]
Hello All,

The title speak by himself :D Did someone succed to deploy Kyoo on K3S ? 

To be honnest, i've not tried yet  but my homelab is hosted on 3 K3s nodes, Jellyfin is deployed but i'm really interested in Kyoo and the more i read about it the more i'm tempted about give it a try.
If someone already deployed it, what were the hard points an other things to focus on ? 

thx :)

**Comment by zoriya** [comment]
Yes, some folk deployed it on k8s! @acelinkio has made a helm chart available at https://github.com/zoriya/Kyoo/tree/master/chart. An app-template chart is also available on the wild. You can hop into the `Home-Operation` discord server if you want, there are multiple folks using kyoo on k8s there!

**Reply by Alulu2401** [reply]
Ow great you made my day ;) 
thanks a lot :)
